version/1/4/heb/code_of_conduct.md

```diff
@@ -0,0 +1,65 @@
+# קוד התנהגות לתורם
+
+## ההתחייבות שלנו
+
+לטובת עידוד של סביבה פתוחה ומקבלת, אנחנו בתור התורמים והמתחזקים מתחייבים להפוך 
+את ההשתתפות בפרויקט שלנו ואת הקהילה שלנו לחוויה נטולת הטרדות לכל אדם באשר הוא
+ללא הבדלי גיל, משקל, מגבלות פיזיות, מוצא אתני, הזדהות מגדרית, רמת ניסיון, לאום, 
+מראה חיצוני, גזע, דת, זהות ונטיה מינית.
+
+## הסטנדרטים שלנו
+
+דוגמאות של התנהגות התורמת לסביבה חיובית כוללות:
+
+* שימוש בשפה מקבלת ומברכת
+* כיבוד של נקודות מבט וחוויות שונות
+* קבלה של ביקורת בהבנה
+* התמקדות במה שהכי טוב לקהילה
+* הפגנת אמפתיה כלפי חברים אחרים מהקהילה
+
+דוגמאות של התנהגות לא מקובלת כוללות:
+
+* שימוש בשפה מינית, תמונות גסות, או הטרדה מינית
+* הטרלה, תגובות מעליבות/משפילות, תקיפה אישית או פוליטית
+* הטרדה פומבית או אישית
+* פרסום מידע אישי של אחרים, כגון כתובת פיזית או אלקטרונית, בלי אישור מפורש
+* התנהגות כלשהי שאינה סבירה בסביבת עבודה מקצועית
+
+## האחריויות שלנו
+
+מתחזקי הפרויקט אחראיים להבהיר את הסטנדרטים להתנהגות מקובלת ומצופים לפעול באופן 
+מתאים והוגן בתגובה לכל אירוע של התנהגות בלתי מקובלת.
+
+יש למתחזקי הפרויקט את הזכות והאחריות להסיר, לערך, או לדחות תגובות, קומיטים, קוד, 
+עדכוני וויקי, נושאים, או תרומות אחרות שאינם מיושרות עם קוד ההתנהגות הנ״ל, או לחסום
+באופן זמני או קבוע כל תורם להתנהגויות אחרות שהם מוצאים אי הולמות, מאיימות, פוגעניות,
+או פוגעות.
+
+
+## היקף
+
+קוד ההתנהגות הנ״ל תקף גם במרחבי הפרויקט וגם במרחבים ציבוריים כאשר אינדיבידואל מייצג
+את הפרויקט או את הקהילה שלו. דוגמאות לייצוג פרויקט או קהילה כוללות שימוש בכתובת 
+אימייל רשמית של הפרויקט, פרסום דרך חשבון מדיה חברתית שלו, או התייצבות כנציג ממונה
+במפגש באינטרנט או באופן לא מקוון. ייתכן שהייצוג של הפרויקט יורחב ויובהר על ידי מתחזקיו.
+
+## אכיפה
+
+
+תופעות של התנהגות מעליבה, מטרידה, או לא מקובלת יכולות להיות מדווחות על ידי יצירת קשר
+עם הקבוצה של הפרויקט ב [הכנס כתבות אימייל]. כלל התלונות יבחנו ויחקרו ויענו על ידי 
```

Comment:
```
תופעות של התנהגות מעליבה, מטרידה, או לא מקובלת יכולות להיות מדווחות על ידי יצירת קשר
עם הקבוצה של הפרויקט ב
```

better phrasing:
```
ניתן לדווח על תופעות של התנהגות מעליבה, מטרידה, או לא מקובלת על ידי יצירת קשר
עם הקבוצה של הפרויקט ב 
```

In free translation, this changes "Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at" to "One may report instances of abusive, harassing, or otherwise unacceptable behavior by contacting the project team at". It reads better in Hebrew.